Thorium-Sim/thorium-nova
# Flight director flex layout sync

## Description

Makes it possible to sync Flight Director flex layouts with Thoriumsim.com accounts.

## Related Issue








Closes #278 

## How do you know the changes work correctly?

Manual Test:
- Create a Thoriumsim.com account
- Sign into the ThoriumSim.com account in Thorium Nova
- Start a flight
- Assign your client to the flight director station
- There should be a menu at the top with options to clear the flex layout and save the flex layout.
- Save the flex layout. It should appear as an option in the menu
- Reload the browser and open the saved flex layout menu. The flex layout you saved should still be there.
- Log out of your Thorium account. The saved flex layout menu should disappear.

## Screenshots (if appropriate):
<img width="520" alt="Screen Shot 2022-04-23 at 10 08 51 AM" src="https://user-images.githubusercontent.com/6558157/164909483-edc8633e-9249-440e-bac2-0d67937132ec.png">

## Types of changes



      change)

## Checklist:
